.version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
@faizanahmad055 change the version to 0.0.1 as we can not call it a 1.0.0 release yet